app/lib/session.coffee

```diff
@@ -1,29 +1,17 @@
-storage = window.sessionStorage ? window.localStorage
```

Comment:
This feature is supposed to be a session id, which refreshes just like a browser session. If researchers are using session ids to analyze anonymous users, then don't use localStorage as a backup.

See https://github.com/zooniverse/front-end-monorepo/pull/6499/files#r1861103668